### DISCUSSION TITLE
Couldn't connect to Discord's IPC socket

### DISCUSSION THREAD

**Discussion by din-dawn** [Radiicall/jellyfin-rpc#96] [Q&A] [answered]
I installed Jellyfin-RPC on fedora in Synology's DSM Docker, but this error comes out when I try to run it. There is no error when I use the same main.json config file with windows version. I am new to linux so I wonder if I set anything wrong or it is synology's issue. Thank you.
Jellyfin-RPC 0.14.0
fedora 39 on Synology docker
Jellyfin server version 10.8.13
DS720+, system version DSM 7.2.1-69057 Update 1
![image](https://github.com/Radiicall/jellyfin-rpc/assets/56496024/ee688537-64aa-4cc3-991c-24959df51ef8)

**Comment by justradical** [comment, marked as answer]
You need to have discord running on the device running jellyfin-rpc

**Comment by prochy-exe** [comment]
if you are bored and still wanna get it working, this might help, not sure about containers but it works for me on a raspberry pi 2b https://github.com/Pr0chy/jellyfin-rpc-discord-bridge
